carball/analysis/stats/rumble/rumble.py

```diff
@@ -0,0 +1,183 @@
+import math
+from typing import Dict, List
+
+import pandas as pd
+
+from carball.generated.api import game_pb2
+from carball.generated.api.player_pb2 import Player
+from carball.generated.api.stats.player_stats_pb2 import PlayerStats
+from carball.generated.api.stats.team_stats_pb2 import TeamStats
+from carball.generated.api.stats.events_pb2 import RumbleItemEvent
+from carball.generated.api.stats.rumble_pb2 import RumbleItems
+from carball.json_parser.game import Game
+from carball.analysis.stats.stats import BaseStat
+from carball.generated.api.metadata.game_metadata_pb2 import RANKED_RUMBLE, UNRANKED_RUMBLE
+
+POWER_UPS = [
+    'ball_freeze',
```

Comment:
If you create an enum in proto you can avoid this list and use that instead.

Reply:
make sense, thanks!